component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/emailotp/EmailOTPAuthenticatorConstants.java

```diff
@@ -127,6 +127,13 @@
     public static final String SERVICE_PROVIDER_NAME = "serviceProviderName";
     public static final String IP_ADDRESS = "ipAddress";
 
+   public static final String USER_AGENT = "user-agent";
```

Comment:
formatting issue

Reply:
Fixed in https://github.com/wso2-extensions/identity-outbound-auth-email-otp/pull/94/commits/5a5d198b3054bf710058f1c945a1f73b350bd783

Reply:
Fixed in https://github.com/wso2-extensions/identity-outbound-auth-email-otp/pull/94/commits/5a5d198b3054bf710058f1c945a1f73b350bd783